client/asset/interface.go

```diff
@@ -319,6 +319,8 @@ type Wallet interface {
 	// EstimateRegistrationTxFee returns an estimate for the tx fee needed to
 	// pay the registration fee using the provided feeRate.
 	EstimateRegistrationTxFee(feeRate uint64) uint64
+	// FallbackFeeRate returns the fallback fee rate.
+	FallbackFeeRate() uint64
```

Comment:
I think we should flip the script on this, and instead add a method like 
```Go
EstimateWithdrawalFees(amt, feeSuggestion uint64) (fees, feeRate uint64, err error)
```
This would allow the wallet to use whatever internal methods it already uses to find and apply a fee rate, and you'd get an accurate value for total fees (that considers available inputs for utxo-based). It's also a similar pattern to that already used for estimating swaps, redeems, and the registration fee, so fits the interface nicely.